system/ee/legacy/helpers/multibyte_helper.php

```diff
@@ -13,16 +13,53 @@
  * Multibyte Helpers
  */
 
+if ( ! function_exists( 'ee_get_encoding' ) )
+{
+	function ee_get_encoding($encoding)
+	{
+
+		if (null === $encoding) {
+
```

Comment:
This isn't following EE's coding standard. Although I'd recommend new stuff follow PSR-1, EE's standard would be:

`
	function ee_get_encoding($encoding)
	{
        if (null === $encoding)
        {
            return ee()->config->item('charset') ?: 'utf8';
        }
        ...
    }
`

Basically remove the extra spaces and move the brackets to a new line.

Reply:
Updated in latest commits.

Reply:
I prefer the variable order the original way, it prevents accidental assignment `if ($encoding = null)` will not throw an error but is incorrect whereas `if (null = $encoding)` throws a parse error.